bun/whatsapp-with-vonage/src/main.ts

```diff
@@ -0,0 +1,65 @@
+import jwt from "jsonwebtoken";
+import { fetch } from "undici";
+import { getStaticFiles } from "./utils";
+
+export default async ({ req, res, log, error }) => {
+  if (req.method === "GET") {
+    return res.send(getStaticFiles, 200, {
+      "Content-Type": "text/html; charset=utf-8",
+    });
+  }
+
+  const hasher = new Bun.CryptoHasher("sha256");
+  const hashedBuffer = hasher.update(JSON.stringify(req.body)).digest("hex");
+  const hashedValue = hashedBuffer.toString();
+
+  const authHeader = req.headers.authorization.split(" ")[1];
+  const decodedToken = jwt.verify(authHeader, Bun.env.VONAGE_SIGNATURE_SECRET, {
+    algorithms: ["HS256"],
+  });
+
+  if (hashedValue != decodedToken["payload_hash"]) {
+    return res.json(
+      {
+        ok: false,
+        error: "Payload hash mismatched",
+      },
+      401
+    );
+  }
+
+  const secret = `${Bun.env.VONAGE_API_KEY}:${Bun.env.VONAGE_ACCOUNT_SECRET}`;
```

Comment:
This variable is unnecessary please match https://github.com/appwrite/templates/blob/6dab35f1f221ba075640d93f5a8e374f4fba4349/node/whatsapp-with-vonage/src/main.js#L35